backend/src/main/kotlin/app/ehrenamtskarte/backend/projects/database/Setup.kt

```diff
@@ -0,0 +1,31 @@
+package app.ehrenamtskarte.backend.projects.database
+
+import app.ehrenamtskarte.backend.config.BackendConfiguration
+import org.jetbrains.exposed.sql.SchemaUtils
+import org.jetbrains.exposed.sql.transactions.transaction
+
+fun setupDatabase(config: BackendConfiguration) {
+    SchemaUtils.create(
+        Projects
+    )
+
+    transaction {
+        val dbProjects = ProjectEntity.all()
+
+        // Delete projects not existing anymore in database
+        dbProjects.forEach { dbProject ->
+            if (config.projects.none { it.id == dbProject.project }) {
+                dbProject.delete()
+            }
+        }
```

Comment:
I would say that we should not delete them, but let them be ophans in the DB :D

Else its possible that data gets deleted if I change/malformat the config.